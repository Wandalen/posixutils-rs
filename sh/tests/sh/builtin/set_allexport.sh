set -a
echo $?

test_var_a=a
echo $test_var_a
awk 'BEGIN { print ENVIRON["test_var_a"] }'

echo $((test_var_b=1234))
echo $test_var_b
awk 'BEGIN { print ENVIRON["test_var_b"] }'

echo ${test_var_c=c}
echo $test_var_c
awk 'BEGIN { print ENVIRON["test_var_c"] }'

set +a

test_var_d=d
echo $test_var_d
awk 'BEGIN { print ENVIRON["test_var_d"] }'
