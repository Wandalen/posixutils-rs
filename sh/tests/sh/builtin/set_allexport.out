0
a
a
1234
1234
1234
c
c
c
d

